### PR TITLE
FOGL-363 omf_translator - Working stage - improving error handling / added ret…

### PR DESCRIPTION
**Pull request by sstefano69**
…ry on OMF / configuration channelID

### CODE REVIEW

File: src/python/foglamp/translators/omf_translator.py [hunk 1]
```diff
@@ -650,54 +621,48 @@ def send_omf_message_to_end_point(message_type, omf_data):
...
-
-        _log.addHandler(handler)
+        else:
+            loop_continue = False
 
-    except Exception as e:
-        raise Exception(_message_list["e000005"].format(e))
+    if status != 0:
+        _logger.warning(message)
+        raise Exception(message)
```
Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: Code is running but not efficient, We need to improve this piece of code and remove un-necessary code and variables (loop_continue, status, max_retry):
   ```
   num_retry = 1 
   while num_retry < 5:
        try:
            do_something()
            break
        except Exception as e:
            sleep()
            num_retry += 1
```

File: src/python/foglamp/translators/omf_translator.py [hunk 2]
```diff
@@ -896,88 +875,129 @@ def omf_object_creation():
...
-                            omf_object_creation()
+                                # Updates latest position sent
+                                new_position = db_row.id
 
-                            debug_msg_write("INFO", "db row |{0}| |{1}| |{2}| ".format(db_row.id,
-                                                                                       db_row.user_ts,
-                                                                                       db_row.reading))
+                            except Exception as e:
+                                # Updates statistics
+                                _num_unsent += 1
```
Comment (sstefano69) [inline src/python/foglamp/translators/omf_translator.py]: this _num_unsent is for internal reason it is not related (requested) for the statistics.

**Comment by sstefano69** [comment]
@vaibhav-ScaleDB  - I changed the code of send_omf_message_to_end_point,
as the code changed for a Ivan request  we maybe close this one and work on new PR

File: src/python/foglamp/translators/omf_translator.py [hunk 3]
```diff
@@ -650,56 +622,47 @@ def send_omf_message_to_end_point(message_type, omf_data):
...
-    global _log
+    global _omf_max_retry
 
-    try:
-        msg_header = {'producertoken': _producer_token,
-                      'messagetype':   message_type,
-                      'action':        'create',
-                      'messageformat': 'JSON',
-                      'omfversion':    '1.0'}
+    sleep_time = _omf_retry_sleep_time
```
Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: _omf_retry_sleep_time is undefined variable. Its not defined and not initialised with a default value.

**Comment by sstefano69** [comment]
is is going to be closed favor of fogl-366 - #109

**Comment by sstefano69** [comment]
closed in favor of fogl-366 - #109

**Pull request closed without merging**